_posts/2017-08-31-Rust-1.20.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: post
+title: "Announcing Rust 1.20"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.20.0. Rust
+is a fast, fearless, and friendly programming language.
+
+If you have a previous version of Rust installed, getting Rust 1.20 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.20.0][notes] on GitHub. 1443 patches were landed in this release.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1200-2017-08-31
+
+### What's in 1.20.0 stable
+
+In Rust, you can define traits, structs, and enums that have "associated functions":
```

Comment:
Maybe "you can already define"? That way it's clear this isn't a new feature.

Reply:
Oh yeah, that confused me as well.